build/utils/cli.js

```diff
@@ -39,6 +39,23 @@ const fetchFilesFromPath = (location) => {
         });
 };
 
+const fetchDirectoriesFromPath = (location) => {
+    const directory = fs.readdirSync(location);
+    const directories = [];
+    directory.map(file => {
+        if (fs.statSync(path.join(location, file)).isDirectory()) {
+            fetchDirectoriesFromPath(path.join(location, file));
```

Comment:
@easteregg What is the purpose of calling the function recursively without using its result?